chandra_aca/tests/test_aca_image.py

```diff
@@ -524,8 +524,8 @@ def images_check_range(start, stop, img_table, *, bgsub):
             row8x8 = row["IMGROW0_8X8"]
             col8x8 = row["IMGCOL0_8X8"]
 
-            # If row8x8 or col8x8 is < 0 just skip the test for now
-            if row8x8 < 0 or col8x8 < 0:
+            # If row8x8 or col8x8 is < -512 just skip the test for now
+            if row8x8 < -512 or col8x8 < -512:
```

Comment:
And I have an update to handle the "off the negative CCD edges", but I don't think that we can get real data like that at all, so maybe there is no point.